packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -168,6 +168,17 @@ export function getInternalReactConstants(
     NoPriority: 90,
   };
 
+  if (gte(version, '17.0.2-alpha')) {
```

Comment:
I think the best way to fix this would have been to bump package versions along with the commit and then do `gt(version, '17.0.2')` because 17.0.2 was published already before this change, so this technically breaks DevTools for that release right? (Also I don't think we did an alpha of 17.0.2 did we?)

I'm off today so I probably won't follow up on this just FYI but I saw the comment :D

Reply:
Cool, thanks! I'll do that